### ISSUE TITLE
Type info of a selection throws an error

### ISSUE THREAD

**Issue by kRITZCREEK** [atom-haskell/haskell-ghc-mod#12]
Hovering over a selection yields this error: 

```
TypeError: Cannot read property 'row' of undefined
  at /home/creek/.atom/packages/haskell-ghc-mod/lib/ghc-modi-process.coffee:198:37
  at /home/creek/.atom/packages/haskell-ghc-mod/lib/ghc-modi-process.coffee:189:9
  at /usr/share/atom/resources/app.asar/node_modules/temp/lib/temp.js:252:7
  at FSReqWrap.oncomplete (fs.js:77:15)
```

**Comment by lierdakil** [comment]
Thank you very much for reporting. This is actually a problem with ide-haskell. Regardless, just pushed a fix, ide-haskell-0.5.1 should work.

**Comment by kRITZCREEK** [comment]
Thx a bunch!
